okhttp/src/main/java/okhttp3/EventListener.java

```diff
@@ -193,11 +193,10 @@ public void requestHeadersStart(Call call) {
    *
    * <p>This method is always invoked after {@link #requestHeadersStart(Call)}.
    *
-   * <p>{@code throwable} will be null in the case of a successful attempt to send the headers.
-   *
-   * <p>{@code throwable} will be non-null in the case of a failed attempt to send the headers.
+   * @param headerLength the length in java characters of headers to be written.
```

Comment:
I think we want to say “in bytes, prior to any encoding like HPACK” . . . because headers are ASCII-only, Java characters and bytes will be the same thing. And I like keeping the units for headers and bodies interchangeable.

Reply:
... so you can add ’em up!